addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -142,12 +150,48 @@ export class HtmlField extends Component {
         return this.props.readonly && this.props.cssReadonlyAssetId;
     }
     get wysiwygOptions() {
+        let dynamicplaceholderOptions = {};
+        if (this.props.dynamicPlaceholder) {
+            dynamicplaceholderOptions = {
+                // Add the powerbox option to open the Dynamic Placeholder
+                // generator.
+                powerboxCommands: [
+                    {
+                        category: this.env._t('Marketing Tools'),
```

Comment:
I don't understand. What is the link between this marketing powerbox command and the options of the dynamic placeholder ?

Reply:
Discussed IRL.